sandbox/gas/tube/go

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python3.5
```

Comment:
SOLVCON should eventually be compatible to Python 3.6.  We should be generic for Python 3 in driving script.